jwst/tweakreg/tweakreg_step.py

```diff
@@ -279,35 +281,34 @@ def process(self, input):
             g = grp_img[0]
             if len(g) == 0:
                 raise AssertionError("Logical error in the pipeline code.")
-            group_name = _common_name(g)
+            group_name = g[0].meta.group_id
             imcats = list(map(self._imodel2wcsim, g))
```

Comment:
Thanks! Since this is now using `model.meta.group_id`, how about assigning the `group_id` for the corrector when it's created (and the other meta data is assigned)?
https://github.com/spacetelescope/jwst/blob/9429dc65e509bec07b7912c5e6f228257d1cfacf/jwst/tweakreg/tweakreg_step.py#L604-L611

Reply:
Ah, yes! Just a sec